client/extensions/woocommerce/app/promotions/promotions-list-row.js

```diff
@@ -47,13 +47,13 @@ function getTimeframeText( promotion, translate, moment ) {
 		} );
 	}
 	if ( promotion.startDate ) {
-		return translate( '%(startDate)s - No expiration date', {
+		return translate( '%(startDate)s - No end date', {
 			args: {
 				startDate: moment( promotion.startDate ).format( 'll' ),
 			},
 		} );
 	}
-	return translate( 'No expiration date' );
+	return translate( 'No end date' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **78** times:
`translate( 'End Date' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/promotion-list-search)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).